src/ts/ts-genapi/export.ts

```diff
@@ -0,0 +1,356 @@
+import {
+    ApiModel,
+    ApiItem,
+    ApiItemKind,
+    ApiDeclaredItem,
+    ExcerptTokenKind
+  } from '@microsoft/api-extractor-model';
+
+import {writeFile} from 'fs';
+
+const jsTokens = require("js-tokens");
+
+class TokensBuilder
```

Comment:
there's a lot going on in this file -- is it possible to break it into modules and put `TokensBuilder` in its own file at least?